UnityProject/Assets/Scripts/HealthV2/Living/CirculatorySystem/CirculatorySystemBase.cs

```diff
@@ -204,6 +205,30 @@ public void BloodSaturationCalculations(float HeartEfficiency)
 			{
 				foreach (var KVP in bloodAndValues.Value)
 				{
+					var PurityMultiplier = 1f;
+
+					var BloodPressure = 1f;
+
+					var PercentageBloodPressure = BloodPool.Total  / StartingBlood;
+					if (PercentageBloodPressure < 0.75f)
+					{
+						BloodPressure = PercentageBloodPressure / 0.75f;
```

Comment:
I don't undertand this change

Reply:
If you got 0.001 you should probably not be alive

Reply:
elaborate. I'm cronically dumb

Reply:
If everything in your blood vessels consisted of 1u of blood then that's not enough for your heart to pump, and therefore not enough for your organs to receive proper oxygenation

Reply:
So what does dividing by that amount do?

Reply:
so, if it's below 0.75 it divides by 0.75

so, let's say 0.5 is helpful your blood body is


0.5 / 0.75 = 0.666 equals the multiplier to apply to oxygen saturation
basically the idea is
0.75 / 0.75 = 1
0  / 0.75 = 0